prisma/schema.prisma

```diff
@@ -136,6 +149,7 @@ model User {
   submissionsReviewed    Submission[] @relation("userReviewedSubmissions")
   submissions            Submission[] @relation("userSubmissions")
   userLessons            UserLesson[]
+  comments               Comment[]
```

Comment:
You might have forgotten this here. There's no `UserLesson <--> Comment` relation on the model.

Reply:
There is one to many User-Comment relation. Line 152 is from `User` model and not from `UserLesson`.

Reply:
Oh right, sorry.